### ISSUE TITLE
Extension creates 7 XML scratchpads upon install

### ISSUE THREAD

**Issue by kAlvaro** [buenon/scratchpads#4]
I presume it's a leftover from some tests:

```
    Directorio: C:\Users\alvaro\.vscode\extensions\buenon.scratchpads-0.0.3\scratchpads\afec1e30600b4ee898f7bf225c38ba44


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----       15/07/2018     10:51              0 scratch.xml
-a----       15/07/2018     10:51              0 scratch1.xml
-a----       15/07/2018     10:51              0 scratch2.xml
-a----       15/07/2018     10:51              0 scratch3.xml
-a----       15/07/2018     10:51              5 scratch4.xml
-a----       15/07/2018     10:51              0 scratch5.xml
-a----       15/07/2018     10:51              0 scratch6.xml
```

**Comment by buenon** [comment]
Correct.
Probably didn't clean the folder before running package.

**Comment by buenon** [comment]
Fixed in version 0.0.5
